app/models/controlled_work_document_content.rb

```diff
@@ -154,15 +164,24 @@ def combined_childcare_costs
       (session_data.dig("api_response", "assessment", "partner_disposable_income", "childcare_allowance") || 0)
   end
 
-  def additional_non_smod_properties_sum(attribute)
-    additional_non_smod_properties.sum { _1[attribute] }
+  def additional_properties_sum(attribute, *modifier)
+    additional_properties(*modifier).sum { _1[attribute] }
   end
 
-  def additional_non_smod_properties
-    @additional_non_smod_properties ||= [
-      (session_data.dig("api_response", "assessment", "capital", "capital_items", "properties", "additional_properties") unless additional_property_in_dispute?),
-      session_data.dig("api_response", "assessment", "partner_capital", "capital_items", "properties", "additional_properties"),
-    ].flatten.compact
+  def additional_properties(property_type = "all")
+    raise "Invalid modifier in YML" unless VALID_PROPERTY_TYPES.include?(property_type)
+
+    @additional_properties ||= if property_type.include?("non_smod")
```

Comment:
Given that we're now passing in a property_type, we shouldn't memoize (i.e. use `@additional_properties ||= `) any more. Because if first I called `additional_properties("all")` it would assign an array of all additional properties to `@additional_properties`, and if I then called `additional_properties("non_smod")`, all additional properties would still be returned to me.

Reply:
Also, arguably the check we want on the right hand side is `if property_type == "non_smod"` because if, as an absurd example, we added `smod_and_non_smod` to VALID_PROPERTY_TYPES, this "include" would break.

Reply:
will sort these two things out now :)